site-src/api-types/backendtlspolicy.md

```diff
@@ -0,0 +1,133 @@
+this API object allows users to specifically configure **backend** TLS termination.  For more information on TLS
+configuration in Gateway API, see [TLS Configuration](/guides/tls/).
+
+![Image showing three Termination Types](/images/tls-termination-types.png)
+
+BackendTLSPolicy is a Direct [PolicyAttachment](/reference/policy-attachment/) without defaults or overrides,
+applied to a Service that accesses a backend, where the BackendTLSPolicy resides in the same namespace as the
+Service to which it is applied. The BackendTLSPolicy and the Service must reside in the same namespace in order
+to prevent the complications involved with sharing trust across namespace boundaries.  
+
+One can reuse the same BackendTLSPolicy for all the different Routes that might point to a referenced Service.
+
+## Spec
+
+The specification of a [BackendTLSPolicy][backendtlspolicy] consists of:
+
+- [TargetRef][targetRef] - Defines the targeted API object of the policy.  Only Service is allowed.
+- [TLS][tls] - Defines the configuration for TLS, including hostname, CACertRefs, and WellKnownCACerts.
+- [Hostname][hostname] - Defines the Server Name Indication (SNI) that the Gateway uses to connect to the backend.
+- [CACertRefs][caCertRefs] - Defines one or more references to objects that contain PEM-encoded TLS certificates,
+which are used to establish a TLS handshake between the Gateway and backend Pod.  Either CACertRefs or WellKnownCACerts
+may be specified, but not both.
+- [WellKnownCACerts][wellKnownCACerts] - Specifies whether system CA certificates may be used in the TLS
+handshake between the Gateway and backend Pod.  Either CACertRefs or WellKnownCACerts may be specified, but not both.
+
+![Image showing BackendTLSPolicy objects](images/backendtlspolicy-api.png)
+
+The following illustrates a BackendTLSPolicy that configures TLS for a Service serving a backend:
+![HTTPRoute with backendTLSPolicy](/images/httproute-with-backend-tls-policy.png)
+
+```mermaid
+flowchart LR
+    client(["client"])
+    gateway["Gateway"]
+    style gateway fill:#02f,color:#fff
+    httproute["HTTP<BR>Route"]
+    style httproute fill:#02f,color:#fff
+    service["Service"]
+    style service fill:#02f,color:#fff
+    pod1["Pod"]
+    style pod1 fill:#02f,color:#fff
+    pod2["Pod"]
+    style pod2 fill:#02f,color:#fff
+    client -.->|HTTP <br> request| gateway
+    gateway --> httproute
+    httproute -.->|BackendTLSPolicy|service
+    service --> pod1 & pod2
+```
+
+### Targeting backends
+
+A BackendTLSPolicy targets a backend Pod (or set of Pods) via a`TargetRef` to a Service.  This TargetRef is a
+required object reference that specifies a Service by its Name, Kind (Service), and optionally its Namespace and Group.
+TargetRef identifies the Service for which your HTTPRoute requires TLS.
+
+!!! info "Restrictions"
+- Cross-namespace certificate references are not allowed.
```

Comment:
Hard to tell without working deploy preview, but I think you may need a new line + indentation here to match what you have below. 
```suggestion

    - Cross-namespace certificate references are not allowed.
```